{
  log.info("Expected exception: " + e);
  assertNotNull(e.reconnectFuture());
  e.reconnectFuture().get();
}
